Formula/h/hdf5.rb

```diff
@@ -7,63 +7,66 @@ class Hdf5 < Formula
   revision 1
   version_scheme 1
 
-  # This regex isn't matching filenames within href attributes (as we normally
-  # do on HTML pages) because this page uses JavaScript to handle the download
-  # buttons and the HTML doesn't contain the related URLs.
   livecheck do
-    url "https://www.hdfgroup.org/downloads/hdf5/source-code/"
-    regex(/>\s*hdf5[._-]v?(\d+(?:\.\d+)+)(?:-\d+)?\.t/i)
+    formula "hdf5"
```

Comment:
It can't use itself to livecheck. Please keep the original

Reply:
Done!